sql/17.0/extra/returning.sql

```diff
@@ -0,0 +1,146 @@
+--Testcase 07:
+UPDATE "type_STRING" SET col = '_' || substr(col, 2) RETURNING *;
+--Testcase 08:
+SELECT * FROM "type_STRING";
+--Testcase 09:
+ EXPLAIN (VERBOSE, COSTS OFF)
+DELETE FROM "type_STRING" RETURNING *;
+--Testcase 10:
+DELETE FROM "type_STRING" RETURNING *;
+
+--Testcase 11:
+ EXPLAIN (VERBOSE, COSTS OFF)
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
+--Testcase 12:
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
+
+--Testcase 13:
+SELECT * FROM "type_BYTE";
+--Testcase 14:
+ EXPLAIN (VERBOSE, COSTS OFF)
+DELETE FROM "type_BYTE" RETURNING *;
+--Testcase 15:
+DELETE FROM "type_BYTE" RETURNING *;
+--Testcase 16:
+SELECT * FROM "type_SINT";
+--Testcase 17:
+DELETE FROM "type_SINT" RETURNING *;
+--Testcase 18:
+SELECT * FROM "type_BINT";
+--Testcase 19:
+DELETE FROM "type_BINT" RETURNING *;
+--Testcase 20:
+SELECT * FROM "type_INTEGER";
+--Testcase 21:
+DELETE FROM "type_INTEGER" RETURNING *;
+--Testcase 22:
+SELECT * FROM "type_FLOAT";
+--Testcase 23:
+DELETE FROM "type_FLOAT" RETURNING *;
+--Testcase 24:
+SELECT * FROM "type_DOUBLE";
+--Testcase 25:
+DELETE FROM "type_DOUBLE" RETURNING *;
+--
+set datestyle=ISO;
+--Testcase 26:
+SELECT * FROM "type_TIMESTAMP";
+--Testcase 27:
+DELETE FROM "type_TIMESTAMP" RETURNING *;
+--Testcase 28:
+SELECT * FROM "type_BLOB";
+--Testcase 29:
+DELETE FROM "type_BLOB" RETURNING *;
+--Testcase 30:
+ALTER TABLE "type_UUID" ALTER COLUMN "i" OPTIONS (ADD key 'true');
+--Testcase 31:
+DELETE FROM "type_UUID" RETURNING *;
+--Testcase 32:
+DELETE FROM "type_BITSTRING" RETURNING *;
+
+--Testcase 33:
+SELECT * FROM typetest;
+--Testcase 34:
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
+--Testcase 35:
+ EXPLAIN (VERBOSE, COSTS OFF)
+INSERT INTO "type_STRING"(col) VALUES ('string') RETURNING *;
+--Testcase 36:
+INSERT INTO "type_BYTE"(col) VALUES ('c') RETURNING *;
+--Testcase 37:
+INSERT INTO "type_SINT"(col) VALUES (32767) RETURNING *;
+--Testcase 38:
+INSERT INTO "type_SINT"(col) VALUES (-32768) RETURNING *;
+--Testcase 39:
+INSERT INTO "type_BINT"(col) VALUES (9223372036854775807) RETURNING *;
+--Testcase 40:
+INSERT INTO "type_BINT"(col) VALUES (-9223372036854775808) RETURNING *;
+--Testcase 41:
+INSERT INTO "type_INTEGER"(col) VALUES (9223372036854775807) RETURNING *;
+
+--Testcase 42:
+INSERT INTO "type_FLOAT"(col) VALUES (3.1415) RETURNING *;
+--Testcase 43:
+INSERT INTO "type_DOUBLE"(col) VALUES (3.14159265) RETURNING *;
+--Testcase 44:
+INSERT INTO "type_TIMESTAMP" VALUES ('2017.11.06 12:34:56.789', '2017.11.06') RETURNING *;
+--Testcase 45:
+INSERT INTO "type_TIMESTAMP" VALUES ('2017.11.06 1:3:0', '2017.11.07') RETURNING *;
+--Testcase 46:
+INSERT INTO "type_BLOB"(col) VALUES (bytea('\xDEADBEEF')) RETURNING *;
+--Testcase 47:
+INSERT INTO typetest VALUES(1,'a', 'b', 'c','2017.11.06 12:34:56.789', '2017.11.06 12:34:56.789' ) RETURNING *;
```

Comment:
Could you move this test case to the end of the test file (after testcase 62 and before testcase 47)?

Reply:
Done, please verify.